src/main/java/org/shanerx/tradeshop/util/BukkitVersion.java

```diff
@@ -56,10 +56,14 @@ public int getMinor() {
         return minor;
     }
 
-    public Boolean isBelow19() {
+    public boolean isBelow19() {
```

Comment:
isn't this method redundant? `!isAbove(1.9)` is a valid replacement, no need for duplicates

Reply:
I'll fix it